### PR TITLE
Profile edit

### PR DESCRIPTION
**Pull request by sznowak13**
Erm, not quite even started, but updated profile page layout.

### CODE REVIEW

File: front/src/App.js [hunk 1]
```diff
@@ -1,3 +1,50 @@
...
+//           </div>
+//           <Footer />
+//         </div>
+//       </Router>
+//     );
+//   }
+// }
+//
+// export default App;
+
```
Comment (Ocieplino) [inline front/src/App.js]: Why we need this?

**Review by Ocieplino — CHANGES_REQUESTED** [review]
Delete unused code

**Review by msciwoj23 — APPROVED** [review]

File: front/src/App.js (hunk 1 above)
Comment (sznowak13) [reply front/src/App.js]: I agree, we should delete this, its from the user-profile page branch, cause I checked out this branch from it.

Comment (sznowak13) [reply front/src/App.js]: I pushed changes, please review now.

File: front/src/App.js [hunk 2]
```diff
@@ -58,6 +58,12 @@ import { BrowserRouter as Router, Route } from "react-router-dom";
 import UserProfile from "./sideBard/UserProfile";
 
 class App extends Component {
+  checkAuthorizedRouting() {
```
Comment (pKraszewska) [inline front/src/App.js:14]: i really like this solution, nicely done!

**Review by pKraszewska — APPROVED** [review]

File: front/src/App.js (hunk 2 above)
Comment (sznowak13) [reply front/src/App.js:14]: Thank you (:

**Review by Ocieplino — APPROVED** [review]
now its ok

**Pull request merged**